### ISSUE TITLE
Cholecyst-enteric fistula

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10056]
**Preferred term label:**
Cholecyst-enteric fistula

**Synonyms**

**Definition (free text, please give PubMed ID)**
Fistula between the gallbladder and the intestine

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Gallstone ileus

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
The following is a narrower concept, but probably this is the most likely place such a fistula would form.
[Bilioduodenal fistula](https://hpo.jax.org/browse/term/HP:6000453)
Is there a need for an additional term?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK430834/
Fistulas can form within any part of the GI tract, with approximately 60% occurring in the duodenum due to the proximity. Less commonly, a gallstone may enter the duodenum through the common bile duct, a dilated papilla of Vater, or after an endoscopic sphincterotomy. A fistula between the gallbladder can also occur with the stomach, transverse colon, and distal small bowel.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001043
- label: Cholecyst-enteric fistula
- parents: [Abnormal gallbladder morphology (HP:0012437)](https://hpo.jax.org/app/browse/term/HP:0012437)
- pmid: PMID:28613584
- def: Fistula between the gallbladder and the intestine.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10056
- synonyms:
